int32 sensorID
int32 temperature
int32 light
int32 humidity 